todo_app/app.py

```diff
@@ -1,11 +1,19 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect, url_for
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
-
+@app.route('/items')
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = get_items()
+    return render_template('index.html', items=items)
+
+@app.route('/items/new', methods=['POST'])
+def new_item():
+    title = request.form.get('field_name')
```

Comment:
`field_name` could probably have a more descriptive name!

Reply:
I have updated it to title_name